src/Abc.Zebus/CommandResult.cs

```diff
@@ -1,25 +1,30 @@
-﻿using Abc.Zebus.Util.Extensions;
-using System;
+﻿using System;
 using System.Collections.Generic;
 using System.Linq;
+using Abc.Zebus.Util.Extensions;
 
 namespace Abc.Zebus
 {
     public class CommandResult
     {
+        [Obsolete("Use the constructor with the responseMessage parameter")]
```

Comment:
Will break compatibility for anyone using "Warnings as errors", should not be the case for a minor version.